test/unit/jenkins/chefspec/packages_spec.rb

```diff
@@ -0,0 +1,24 @@
+#
+# jenkins-ci::packages
+#
+require_relative 'spec_helper'
+
+describe 'jenkins-ci::packages' do
+  let(:chef_run) do
+    ChefSpec::Runner.new do |node|
+      env = Chef::Environment.new
+      env.name 'test'
+
+      allow(node).to receive(:chef_environment).and_return(env.name)
+      allow(Chef::Environment).to receive(:load).and_return(env)
+    end.converge(described_recipe)
```

Comment:
These `chef_run` declarations can be reduced to one-liners if you don't need the 'test' environment. (Not sure if you do or not but thought I should mention it.)